admin/RELEASING.md

```diff
@@ -0,0 +1,18 @@
+# How-to release `rustls-platform-verifier`
+
+This document records the steps to publish new versions of the crate since it requires non-trivial preperation and ordering
+that needs to be remembered due to the Android component's distribution.
+
+## Steps
+
+1. Update main crate'a version in `rustls-platform-verifier/Cargo.toml`, and in any additional places.
+2. If any non-test changes have been made to the `android` directory since the last release:
+    1. Update Android artifact version in `android-release-support/Cargo.toml`
+    2. Bump dependency version of the Android support crate in `rustls-platform-verifier/Cargo.toml` to match the new one
+    3. Commit version increase changes on the release branch
+    4. Run `ci/package_android_release.sh` in a UNIX compatible shell
+    5. (Optional) `cargo publish -p rustls-platform-verifier-android --dry-run`
+    6. (Optional) Inspect extracted archive to ensure the local Maven repository artifacts are present
+    7. Publish the Android artifacts' new version: `cargo publish -p rustls-platform-verifier-android`
```

Comment:
@djc @cpu One open question I have here is about the Git tagging/release process: Should we bother attaching the precompiled artifacts to any tag or GitHub release? This would let someone easily download/verify it if they desired. Alternatively, they can just download it from crates.io and do the same thing.

If we do want to include it here, I haven't really come up with a good order to do so.

Reply:
Letting them download from crates.io seems fine for now, can always iterate in response to feedback.

Reply:
That seems reasonable to me. Resolving.